content/blog/2019-01-03-ozunconf18.md

```diff
@@ -0,0 +1,54 @@
+and the excitement was building with the number of issues.
+
+The day before the unconf we ran "Day 0 training" - an afternoon explaining R packages and github. This training is run a little different to a typical course on R packages and github, in that the aim is to get people acquainted with what R packages and GitHub are, that they can learn the material, and that it is worthwhile to learn. We showed people how to create an R pacakge, set it up with version control with git, then put it on github, and share it with others. This afternoon course was developed last year by myself and [Roger Peng](https://twitter.com/rdpeng). This year, [Saras Windecker](https://twitter.com/smwindecker) stepped up to the plate and did an incredible job teaching. We also had a great group of helper instructors, which ended up creating the fantastic teaching ratio of two students for every helper. Everyone picked everything up super quickly, and this meant that we could hit the ground running on Day 1.
+
+[photo of day 0 training]
+
+Day 1 started with a bang, in the beautiful north arts building at the university of Melbourne, and dusted off the tried and tested rOpenSci icebreaker. We then moved on to choose projects. Everyone moved around the room, exploring options, and we settled into groups.
+
+We worked on a variety of projects, ranging from some that we thought might be knocked over in a few hours, like [ozbabynames](https://github.com/ropenscilabs/ozbabynames). (Spoiler alert, it took longer than three hours), to [documenting and learning gganimate](https://github.com/ropenscilabs/learngganimate), [creating paper review toolkits](https://github.com/ropenscilabs/reviewer), shortening / enriching the documenting/reading the documentation process with [tl](https://github.com/ropenscilabs/tl), [automating the creation of vitae's](https://github.com/ropenscilabs/vitae), simplifying the process of recoding free gender text responses with [gendercoder](https://github.com/ropenscilabs/gendercoder), [measuring walkability](https://github.com/ropenscilabs/rwalkable), [making bindertools to recreate and reproduce analysis](https://github.com/ropenscilabs/bindertools), and even [generating synonyms](https://github.com/ropenscilabs/syn). You can see the complete list of projects [here](http://ozunconf18.ropensci.org/projects/).
+
+[photo]
+
+On the second day, [Adam Gruer](https://twitter.com/AdamGruer) led a fireside chat on how to continue the experience of the rOpenSci ozunconf after we wrap up. Among the suggestions were starting or joining coding clubs at your university or place of work. Adam also made the point that a reason so many people enjoy the unconf is that freeing up your time and expectations has the surprising effect of making you more productive, and to try and recreate that idea. Robbie Bonelli had a good way to summarise this:
```

Comment:
revise "fireside chat on how to continue" to indicate it was not to prescribe a how to, but to share ideas e.g. "fireside chat on how we might continue..."